demo/site/src/common/components/breadcrumbs/Breadcrumbs.module.scss

```diff
@@ -0,0 +1,171 @@
+    color: var(--palette-primary-main);
+    text-decoration: none;
+
+    @media (min-width: $breakpoint-sm) {
+        display: none;
+    }
+}
+
+.listContainer {
+    display: none;
+    transform: translateX(100%);
+    transition:
+        transform 0.2s ease-out,
+        visibility 0s linear 0.2s;
+
+    &Expanded {
+        display: block;
+        position: absolute;
+        left: 0;
+        top: 100%;
+        width: 100vw;
+        box-sizing: border-box;
+        z-index: 1;
+        overflow: hidden;
+        background-color: white;
+        height: auto;
+        transform: translateX(0);
+        visibility: visible;
+        transition:
+            transform 0.2s ease-in,
+            visibility 0s linear 0s;
+    }
+
+    @media (min-width: $breakpoint-sm) {
+        display: block;
+        grid-column: 2 / -2;
+        position: static;
+        width: auto;
+        transform: none;
+        transition: none;
+    }
+}
+
+.list {
+    margin: 0;
+    padding: 0;
+    list-style-type: none;
+
+    @media (min-width: $breakpoint-sm) {
+        display: flex;
+        align-items: center;
+        gap: var(--spacing-s200);
+        flex-wrap: wrap;
+        padding: var(--spacing-s400) 0;
+    }
+}
+
+.mobileChevron {
+    flex-shrink: 0;
+    color: var(--palette-gray-700);
+    transition: transform 0.4s ease;
+
+    &Expanded {
+        transform: rotate(180deg);
+    }
+}
+
+.listElement {
+    display: flex;
+    align-items: center;
+    gap: var(--spacing-s200);
+}
+
+.link {
+    color: var(--palette-gray-700);
+    text-decoration: none;
+    text-align: left;
+
+    &Active {
+        color: var(--palette-primary-main);
+    }
+
+    @media (min-width: $breakpoint-sm) {
+        color: var(--palette-gray-700);
+        font-size: 15px;
+
+        &:last-child {
+            color: var(--palette-primary-main);
+        }
+
+        &:hover {
+            text-decoration: underline;
+            color: var(--palette-primary-main);
+        }
+    }
+}
+
+.chevron {
+    display: none;
+
+    @media (min-width: $breakpoint-sm) {
+        display: block;
+        transform: rotate(-90deg);
+        color: var(--palette-gray-400);
+    }
+}
+
+.mobileCornerDownRightIcon {
+    color: var(--palette-gray-700);
+    margin-right: 4px;
+    flex-shrink: 0;
+
+    @media (min-width: $breakpoint-sm) {
+        display: none;
+    }
+}
+
+.mobileBackButton {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    padding: var(--spacing-s500) 0;
+    padding-left: var(--spacing-s600);
+    gap: var(--spacing-s200);
+    align-items: center;
+    border-bottom: 1px solid;
+    color: var(--palette-gray-700);
+}
+
+.linkContainer {
+    display: flex;
+    padding-bottom: var(--spacing-s400);
+    margin-right: 16px;
+
+    &:nth-child(2) {
+        margin-top: var(--spacing-s600);
+    }
+
+    @for $i from 2 through 10 {
+        &:nth-child(#{$i}) {
+            margin-left: 32px * ($i - 1);
+        }
+    }
```

Comment:
This seems a little hacky and won't work if there are more than 10 items... 

Could each of these items be rendered inside a wrapper with a `padding-left`, then all these wrappers are nested within each other so the paddings add up, the more they are nested? 
Or could there be some other generic solution? 🤔